app/views/admin/show.html.erb

```diff
@@ -0,0 +1,56 @@
+<section id="application-<%= @application.id %>">
+  <p>Name: <%= @application.name %></p>
+  <p>Street Address: <%= @application.street_address %></p>
+  <p>City: <%= @application.city %></p>
+  <p>State: <%= @application.state %></p>
+  <p>Zipcode: <%= @application.zip_code %></p>
+  <p>Description: <%= @application.description %></p>
+  <p>Status: <%= @application.status %></p>
+  <p>Pets:</p>
+  <% @application.pets.each do |pet| %>
+    <%= link_to pet.name, "/pets/#{pet.id}" %>
+      <% end %>
+  <% @application.application_pets.each do |app_pet| %>
+    <% if app_pet.status == "Pending" %>
+    <%= form_with(url: "/admin/applications/#{@application.id}", data: {turbo: false}, method: :patch) do |form| %>
+        <%= form.hidden_field :app_pet_id, value: app_pet.pet_id %>
+        <%= form.submit "Approve" %>
+      <% end %>
+    <% elsif app_pet.status == "Approved" %>
```

Comment:
so here we can use .status bc we added status to our db applications_pet table as an attribute. I guess just adding an attribute to the  application pet models file doesn't work?